### PR TITLE
Fix coding standards

### PR DESCRIPTION
**Pull request by PatelUtkarsh**
Follows WordPressVIPMinimum coding standards

### CODE REVIEW

File: public-post-preview.php [hunk 1]
```diff
@@ -649,7 +649,7 @@ private static function verify_nonce( $nonce, $action = -1 ) {
 	 * @return array The post ids. (Empty array if no ids are registered.)
 	 */
 	private static function get_preview_post_ids() {
-		return get_option( 'public_post_preview', array() );
+		return array_map( 'intval', get_option( 'public_post_preview', array() ) );
```
Comment (PatelUtkarsh) [inline public-post-preview.php]: Added intval so strict check in_array doesn't need to convert array to int each time.

Comment (PatelUtkarsh) [reply public-post-preview.php]: Most data being saved is giving post id as int except 1 from `post_updated` hook.
I've updated `unregister_public_preview_on_edit` but we also need to take care of existing serialized data in DB.

File: public-post-preview.php [hunk 2]
```diff
@@ -658,10 +658,10 @@ private static function get_preview_post_ids() {
 	 * @since 2.0.0
 	 *
 	 * @param array $post_ids List of post IDs that have a preview.
-	 * @return array The post ids. (Empty array if no ids are registered.)
+	 * @return bool False if value was not updated and true if value was updated.
 	 */
 	private static function set_preview_post_ids( $post_ids = array() ) {
-		return update_option( 'public_post_preview', $post_ids );
+		return update_option( 'public_post_preview', array_map( 'absint', $post_ids ) );
```
Comment (PatelUtkarsh) [inline public-post-preview.php]: Array map here is technically not needed but just for double check.

**Comment by ocean90** [comment]
Thanks for the PR! I applied the changes manually except for the change to escape translations.

**Pull request closed without merging**